src/components/dashboard/menu-table/popup-forms.js

```diff
@@ -433,11 +435,140 @@ const QRCodeForm = (props) => {
     );
 }
 
+const StyledUploadCSVModal = styled.div`
+    input[type="checkbox"] {
+        display: inline-block;
+    }
+
+    input[type="file"] {
+
+    }
+
+    .file-input {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 200px;
+        margin-bottom: 20px;
+        border: 2px dashed grey;
+        border-radius: 15px;
+
+        &:focus {
+            outline: none;
+        }
+    }
+
+    .error-msg {
+        color: red;
+    }
+    
+
+
+    p {
+        margin-left: 20px;
+        display: inline-block;
+    }
+`
+
+const UploadCSVModal = (props) => {
+    let { show, close, uploadCSV } = props
+    let [overwrite, setOverwrite] = useState(false)
+    let [content, setContent] = useState(null)
+    let [fileName, setFileName] = useState(null)
+    let [errorMessage, setErrorMessage] = useState(null)
+    
+    const onDrop = useCallback(acceptedFiles => {
+        acceptedFiles.forEach((file) => {
+            const reader = new FileReader()      
+            reader.onabort = () => console.error('file reading was aborted')
+            reader.onerror = () => {
+                console.error('file reading has failed')
+                setErrorMessage("Error uploading file")
+            }
+            reader.onload = () => {
+            // Do whatever you want with the file contents
+              const fileContent = reader.result
+              setContent(fileContent)
```

Comment:
I happened to discover that if you uploaded an empty file the view doesn't change or say anything that reminds the user an empty file is uploaded...

Reply:
Empty csv file or just generally empty file?

Reply:
@cchyung An empty csv is what I tried. Not sure about an empty file of ANY format.